compiler/rustc_feature/src/accepted.rs

```diff
@@ -244,6 +244,8 @@ declare_features! (
     (accepted, irrefutable_let_patterns, "1.33.0", Some(44495)),
     /// Allows `#[instruction_set(_)]` attribute.
     (accepted, isa_attribute, "1.67.0", Some(74727)),
+    /// Rescoping temporaries in `if let` to align with Rust 2024.
+    (accepted, if_let_rescope, "1.83.0", Some(124085)),
```

Comment:
Aside from being sorted, this should also probably use `CURRENT RUSTC VERSION`.

Reply:
Applied